sklearn/kernel_approximation.py

```diff
@@ -303,21 +274,26 @@ def transform(self, X, y=None):
         transf = self._transform_sparse if sparse else self._transform_dense
         return transf(X)
 
+    @abstractmethod
+    def _spectrum(self, omega):
+        pass
```

Comment:
I would much rather this be a `NotImplementedError` than be pedantic about code coverage.